src/commons/utils/alerts.py

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+# -*- coding: utf-8 -*-
+# !/usr/bin/python
+"""
+Module for generating email
+"""
+import json
+import logging
+import os
+import smtplib
+import threading
+import time
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.utils import formatdate, COMMASPACE
+
+from config import CORIO_CFG
+from src.commons import commands
+from src.commons.degrade_cluster import get_logical_node
+from src.commons.utils.corio_utils import get_report_file_path
+
+LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=too-few-public-methods
+class Mail:
+    """
+    Module to send mail
+    """
+
+    def __init__(self, sender, receiver):
+        """
+        Init method
+        :param sender: email address of sender
+        :param receiver: email address of receiver
+        """
+        self.mail_host = os.getenv("EMAIL_HOST")
```

Comment:
Please add this in documentation.
EMAIL_HOST And EMAIL_PORT needs to be set as environment variables.

Reply:
Done

Reply:
Done